dvc/remote/ssh/__init__.py

```diff
@@ -47,9 +50,21 @@ class RemoteSSH(RemoteBASE):
     # We use conservative setting of 4 instead to not exhaust max sessions.
     CHECKSUM_JOBS = 4
 
+    # XXX: Friendly reminder to add "reflink" to the default types
+    # after is implemented
+    DEFAULT_CACHE_TYPES = ["symlink", "copy"]
```

Comment:
Actually, how about we leave "copy" by default to provide the same safe behaviour in both local and ssh?

Reply:
@efiop , as discussed previously, this is going to be `reflink, copy`, right?